.azure_pipelines/job_templates/notify-webapp-template.yml

```
@@ -0,0 +1,26 @@
+parameters:
+  name: ''
+  pool: ''
+  customMatrixes: ''
+
+jobs:
+- job: ${{parameters.name}}
+  timeoutInMinutes: 600
+  strategy:
+    matrix:
+      ${{ insert }}: ${{parameters.customMatrixes}}
+  pool:
+    name: ${{ parameters.pool}}
+  steps:
+  # Clone PyTorch Tests repository
+  - bash: |
+      B64_PAT=$(printf "%s"":$_ADOTOKEN" | base64)
```

Can you please explain what are you trying to do here? Do you want to add `:` in front of the token and print it without trailing newline character?
```suggestion
      B64_PAT=$(echo -n ":$_ADOTOKEN" | base64)
```

---

Here I am converting $_ADOTOKEN to Base-64 format. Thank you for your suggestion, it is cleaner and I will update this PR as such.